.github/workflows/publish_nuget.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
     - 'master'
+  pull_request_target:
+    types: [opened, synchronize]
```

Comment:
Why do we use this different notation for this. Maybe also change the rest then.

```yml
  workflow_dispatch:
  push:
    branches: [master]
  pull_request_target:
    types: [opened, synchronize]
  release:
    types: [published]
```

But I think I prefer the `-` notation for readability
```yml
  workflow_dispatch:
  push:
    branches:
    - master
  pull_request_target:
    types:
    - opened
    - synchronize
  release:
    types:
    - published
```

But just pick the one you prefer as long as they all are the same.